subjects/fillit.en.md

```diff
@@ -0,0 +1,109 @@
+## fiil it
+
+### Objectives
+
+Develop a program that receives only one parameter, a txt file which will contain a list of [**Tetrominos**](https://en.wikipedia.org/wiki/Tetromino) to assemble them in order to create the smallest square possible.
+
+### Instructions
+
+All files referring to the project, must be in the same folder.
+
+The program must:
+
+- Be written in **GO**
+- Be compiled to be runned
+- Assemble all of the **Tetrominos** in order to create the smallest square possible
+- Show an error if the input doesn't correspond to a classic **Tetromino** piece
+- Not allow the rotation of the **Tetrominos**
+- Organize the **Tetrominos** by their apparition order in the file
```

Comment:
I suggest we remove this instruction, to allow different implementations and to keep only one objective here : make the smallest square with the tetrominoes.